spec/controllers/api/v1/recipes_controller_spec.rb

```diff
@@ -29,7 +30,8 @@
       {
         recipe: {
           name: 'Leczo',
-          content: 'Very good dish'
+          content: 'Very good dish',
+          video_link: 'https://www.youtube.com/watch?v=60KwNYk09BU'
```

Comment:
Use `let` so you don't have to type the same URL all over again.
```suggestion
          let(:video_link) { 'https://www.youtube.com/watch?v=60KwNYk09BU' }
          # ...
          video_link: video_link
```

Reply:
@Elkorm98 please do as Seb wrote above ☝️